person.rb

```diff
@@ -1,4 +1,5 @@
-class Person
+require './nameable'
+class Person < Nameable
   attr_reader :id
```

Comment:
- [ ] In trying to test creating a new `Person`, I go some errors. Just as the previous reviewer said, kindly make sure the `Person` class is correctly decorated by creating instances of the different classes and testing them if they are actually working. Try the code below to see if everything is working and submit it once more.

- [ ] Test your code against this to see if it works 
```
person = Person.new(22, 'maximilianus')
 person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 capitalizedTrimmedPerson.correct_name
``

## Error:
![image](https://github.com/ghreza-crypto/school-library/assets/50990925/7272415d-109f-47a1-a854-b3f3b12bc3ef)

Reply:
I have tested the code and it is working well, there is no error. You should check it in mian.rb file. There is already a new instance of the person class created.
![test](https://github.com/ghreza-crypto/school-library/assets/53405011/c79fdc09-b53a-4890-ab19-d581c0396192)

Reply:
I have tested the code and it is working well, there is no error. You should check it in mian.rb file. There is already a new instance of the person class created.